### PR TITLE
refactor: replace @edx/paragon and @edx/frontend-build

### PR DESCRIPTION
**Pull request by Mashal-m**
Part of https://github.com/openedx/axim-engineering/issues/23

This replaces the `@edx/paragon` and `@edx/frontend-build` package. 
Imports have been updated to use the same locations in the new package.

### CODE REVIEW

**Comment by jsnwesson** [comment]
Hey @BilalQamar95 and @Mashal-m , just wanted to check in on this PR since it is marked "ready for review"! Who is planned for a code review: AXIM, another eng in Fed-BOM, or an eng in Aperture? This question applies to all of Aperture's MFEs as well (Learner-Dashboard, Learner-Record, Skills). Thanks!

**Comment by BilalQamar95** [comment]
Hi @jsnwesson, generally such PRs are planned to be reviewed by owning squad (aperture in this case). In cases where its someone other than the owning squad we tend to tag them as reviewers. Hope this helps

**Comment by jsnwesson** [comment]
@BilalQamar95 thanks for clarifying! I'll be adding a couple of labels to make it clear that this is ready for review.

File: src/i18n/index.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { messages as headerMessages } from '@edx/frontend-component-header';
 import { messages as footerMessages } from '@edx/frontend-component-footer';
```
Comment (jsnwesson) [inline src/i18n/index.js:2]:
```suggestion
import { messages as headerMessages } from '@openedx/frontend-component-header';
import { messages as footerMessages } from '@openedx/frontend-component-footer';
```
I imagine these will also need to be updated, right?

Comment (jsnwesson) [reply src/i18n/index.js:2]: Ah, I misread the `package.json` changes, which was that only Paragon and Frontend Build are being switched to `@openedx` for this PR. Please disregard 😅

**Review by jsnwesson — APPROVED** [review]
Ran it locally and looks good to me!

**Comment by jsnwesson** [comment]
Ah, looks like there is a package-lock conflict @BilalQamar95 . once that's resolved, I can merge and deploy the PR!

**Comment by BilalQamar95** [comment]
Hi @jsnwesson, I have resolved the conflicts

**Comment by OmarIthawi** [comment]
@BilalQamar95 @jsnwesson this pull request removed the `openedx-atlas` package which is needed for translations:

 - https://github.com/openedx/frontend-app-profile/pull/945/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L35

As far as I can see this was an intentional. I'll open another pull request to get the required package back. Please don't remove, otherwise Tutor MFE builds will be broken.

**Pull request merged**